string ID
uint8 Sticker
string playable
float32 speedCoefficient
Sensor[] sensors
Vector4 position
Vector4 controlCommand
Vector4 modifiedCommand
